### PR TITLE
add crud for cassandra k8s resources

### LINKED ISSUE #23: Create cassandra deployment controller
Due to https://github.com/agalitsyn/yagoda/issues/3 we should config CASSANDRA_SEEDS env variable, it should be list of seed nodes. Our cluster is 1DC, so it should be 3 seeds, but overall amount of nodes can be more in general.

### PR DESCRIPTION
**Pull request by agalitsyn**
Resolves #23 

Confirming that cluster is fully operational better in functional tests, by now I made simple checkers

### CODE REVIEW

File: cassandra-manage.py [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+from pykube.exceptions import KubernetesError
+from pykube.exceptions import HTTPError
+from pykube.exceptions import ObjectDoesNotExist
+
+
+LOG = logging.getLogger('{}:{}'.format(__file__, __name__))
+PROJECT_ROOT = os.path.dirname(os.path.abspath(__file__))
+
+
+def serialize_from_yaml_file(path):
```
Comment (alexey-medvedchikov) [inline cassandra-manage.py]: Grammar-nazi tip: It's de-serialize, actually :)

File: cassandra-manage.py [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+class CassandraService(K8SResource, Service):
+    pass
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Yagoda cassandra '
+                                                 'management tool.')
+    parser.add_argument('action', choices=['create', 'update',
+                                           'delete', 'validate'])
+    group = parser.add_mutually_exclusive_group()
```
Comment (alexey-medvedchikov) [inline cassandra-manage.py:174]: `group = parser.add_mutually_exclusive_group(required=True)`, because there's no default action in main.

Comment (agalitsyn) [inline cassandra-manage.py:174]: default actually presents on line https://github.com/agalitsyn/yagoda/blob/cassandra-controller/cassandra-manage.py#L196

File: cassandra-manage.py (hunk 1 above)
Comment (agalitsyn) [inline cassandra-manage.py]: Right! Sorry.

File: cassandra-manage.py (hunk 2 above)
Comment (alexey-medvedchikov) [inline cassandra-manage.py:174]: doh!

**Pull request merged**